worker/provisioner/lxd-broker.go

```diff
@@ -179,3 +183,28 @@ func (broker *lxdBroker) MaintainInstance(ctx context.ProviderCallContext, args
 	)
 	return err
 }
+
+func (broker *lxdBroker) writeProfiles(machineID string) ([]string, error) {
+	containerTag := names.NewMachineTag(machineID)
+	profileInfo, err := broker.api.GetContainerProfileInfo(containerTag)
+	if err != nil {
+		return nil, err
+	}
+	names := make([]string, len(profileInfo))
+	for i, profile := range profileInfo {
+		err := broker.maybeWriteLXDProfile(profile.Name, &charm.LXDProfile{profile.Config, profile.Description, profile.Devices})
```

Comment:
nit: can we write this as:

```go
..., &charm.LXDProfile{
    Config: profile.Config,
    Description: profile.Description,
    Devices: profile.Devices,
})
```

That way it won't throw a govet info warning around unkeyed fields.

Reply:
done.  i noticed you doing that recently, now i get it.  :-)